components/sidebar.jsx

```diff
@@ -895,11 +916,13 @@ export default class Sidebar extends React.Component {
 
                 <UnreadChannelIndicator
                     show={this.state.showTopUnread}
+                    onClick={this.scrollToFirstUnreadChannel.bind(this)}
```

Comment:
This can just be `onClick={this.scrollToFirstUnreadChannel}`. Since it's defined as an arrow function, `this` is already bound